packages/styles/src/fundamental-styles.scss

```diff
@@ -108,3 +108,4 @@
 
 // BTP Specific components/overwrites
 @import "./btp/avatar.scss";
+@import "skeleton";
```

Comment:
why is this here? Under line 109 are only BTP components overwrites aka have the same names as Fiori

Reply:
this was autogenerated